packages/angular/src/date-time-picker/date-time-picker.component.ts

```diff
@@ -13,61 +13,7 @@ import { format, isThisMinute, subHours, subMinutes } from 'date-fns';
 import { getRangeFromRelative } from './date-time-relative.component';
 import * as languages from 'flatpickr/dist/l10n/index';
 import { I18n } from 'carbon-components-angular/i18n';
-
```

Comment:
Moving the exports changes how this is used (API) and is considered a breaking change. Can't do that in a non-major version.

Re-exporting from here should be ok if you wanna keep it out, as a preparation for the major.